### ISSUE TITLE
Unable to emit from server.

### ISSUE THREAD

**Issue by Cedriking** [deployd/deployd#805]
## Context
Hello, I'm trying to send an emit (broadcast to everyone) when inserting an object to deployd.
I use the server function `Store.insert(obj)` to add an element to deployd and then inside the onPost in deployd I have `emit('create', this);`.

On the client-side I use `dpd.on('create', response => console.log(response));` but I never get anything, how do I emit to everyone once created? Is it because `Store.insert()` doesn't go through the ON POST rules? or is there a bug?

It's important to note that if I send `dpd.articles.post()` (3) request directly from the client-side, it works as expected.

## Expected Behavior
I should get the new inserted object to everyone visiting the site.

## Actual Behavior
The new object is posted, but the emit never happens.

## Possible Fix
I don't have any possible option, tried everything I could.

## Your Environment

* Deployd Version (`deployd -V`): 0.8.9
* Node version (`node -v`): 7.7.4
* Server type and version: GNU/Linux 4.4.0-66-generic x86_64
* Operating System and version: Ubuntu 16.04.2 LTS

**Comment by rgolea** [comment]
Try doing `dpd.on('connection', () => ...)` on your client side. If you don't get that triggered, make sure you are connecting.

**Comment by travisstaloch** [comment]
I am experiencing the same thing.  My browser clients aren't receiving any emit() messages from the server in Chrome whether running in dev environment on windows or on heroku production environment.  I've added `dpd.on('connection', function() {console.log('connection')});` along with my other `dpd.on` functions and not getting any output from them in the console.  I've tried using `dpd.messages.on` too.  I've been working from [https://github.com/NicolasRitouet/deployd-heroku-demo](url).  I have: 
> npm -v: 3.10.10, node -v: v6.11.1, dpd --version: 0.8.9,  socket.io-client@1.7.4,  express@4.15.4

my package.json deps are: 
    "deployd": "^0.8.9",
    "dpd-email": "^0.3.0",
    "dpd-passport": "^0.3.6",
    "express": "^4.15.4"

I see that there are socket polling requests coming from the client but I'm not seeing any type of response.  I thought the problem might be with `server.listen(serverPort, function() {...}` because its showing that `server.address().address` is empty.  I tried using `server.listen(serverPort, 'localhost', function() {...}` which changed server.address().address to 127.0.0.1 but that didn't seem to fix the issue.

Does this question belong in the deployd-users group?  Any feedback is be greatly appreciated.  Thank you.
